app/helpers/sprangular/application_helper.rb

```diff
@@ -12,6 +12,12 @@ def payment_methods
     def js_environment
       config = ::Spree::Config
       store = Spree::Store.current
+      supported_locales = if Object.const_defined?('SpreeI18n')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.